src/System.Data.SqlClient/tests/ManualTests/ProviderAgnostic/ReaderTest/ReaderTest.cs

```diff
@@ -266,5 +270,371 @@ public static void TestMain()
                 }
             }
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        public static async Task TestConcurrentLoadSync()
+        {
+            ConcurrentLoadContext context = new ConcurrentLoadContext(
+                providerFactory: SqlClientFactory.Instance,
+                connectionString: DataTestUtility.TcpConnStr,
+                mode: ConcurrentLoadContext.Mode.Async,
```

Comment:
```suggestion
                mode: ConcurrentLoadContext.Mode.Sync,
```
Passing async mode in sync test ?

Reply:
copy paste error, fixed and retested.